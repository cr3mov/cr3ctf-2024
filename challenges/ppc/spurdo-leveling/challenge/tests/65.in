23770486050597978 944907917033128329
80 803334
79 400
