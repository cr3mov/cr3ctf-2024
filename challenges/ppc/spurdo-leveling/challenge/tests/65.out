29807885403189 1146457130892 299075773277
